app/models/feature.rb

```diff
@@ -1,12 +1,12 @@
 class Feature
-  def self.enabled?(feature_name)
+  def self.enabled?(feature_name:, user: nil)
```

Comment:
```suggestion
  def self.enabled?(feature_name, user: nil)
```

I don't think we need to name the first variable since it _is_ the feature we are switching on.

Reply:
done